handlers/InitiateSalesforceSession.ts

```diff
@@ -79,12 +79,7 @@ export class InitiateSalesforceSession {
         .getSettings()
         .getById('handover_department_name')
     ).value;
-    const rocketChatServerUrl: string = (
-      await this.read
-        .getEnvironmentReader()
-        .getSettings()
-        .getById('rocketchat_server_url')
-    ).value;
+    const rocketChatServerUrl: string = await getServerSettingValue(this.read, 'Site_Url');
```

Comment:
Hitting this when testing
```
url: 'https://livechat.viasatconnect.comapi/v1/login'
```
i guess we need to add a / after places we call the URL, or in the url def here

Reply:
Yup you need to add '/' in the end for every URL in the app configuration.

Reply:
but you removed the url config option :)

Reply:
Ya, I removed it.  But I think in my server it was probably returning a server URL with '/' in the end, since it was working on in my setup. But however, I think I will just add a regex check for '/'.

Reply:
ah yeah our site url setting doesnt have that trailing `/`, yeah checking for that would be a nice feature

Reply:
cool, I will add that check in my next commit 👌